app/javascript/chat/channelRequest.jsx

```diff
@@ -25,7 +25,7 @@ const ChannelRequest = ({ resource: data, handleJoiningRequest }) => (
       {data.channel.status !== 'joining_request' ? (
         <button
           type="button"
-          className="cta"
```

Comment:
There's one more element with this class in that same file, [on line 36](https://github.com/forem/forem/pull/10694/files/cdf6025052217632b446a6427e467db356af0d3c#diff-46793e721c2c259bbbfdefc4a0324054L36).